src/math_extras.rs

```diff
@@ -3,7 +3,7 @@
 // See https://llvm.org/LICENSE.txt for license information.
 // SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
 
-pub(crate) fn align_to_power_of2(value: u64, align: u64) -> u64 {
+pub(crate) const fn align_to_power_of2(value: u64, align: u64) -> u64 {
     assert!(
         align != 0 && (align & (align - 1)) == 0,
         "Align must be a power of 2"
```

Comment:
Pre-existing, but using `let neg_align = align.wrapping_neg()` below should work.